packages/datasets/src/uploadDatasetToHuggingFace.ts

```diff
@@ -0,0 +1,41 @@
+import { uploadFiles } from "@huggingface/hub";
+import { logger } from "mongodb-rag-core";
+
+export interface UploadDatasetToHuggingFaceParams {
+  hf: {
```

Comment:
Any particular reason to use `hf` instead of `huggingface`? Seems like we generally bias to more verbose names.

(if we were writing Swift we could use `🤗` as the field name!)

Reply:
not that i can recall. def prefer verbosity generally. will update.